pkg/api/api.go

```diff
@@ -37,7 +36,7 @@ Powered by open-source software: Etcd(https://etcd.io), Apache License 2.0.
 
 const (
 	// APIPrefix is the prefix of api.
-	APIPrefix = "/apis/v1"
+	APIPrefix = "/apis/v2"
```

Comment:
propose to support both v1 and v2 for some time, or it will suddenly break 3rd party applications.

Reply:
First off, we are on the page where v2 is not compatible to v1 in many ways. We could support v1 prefix but with v2 behavior, which seems weirder to me.